test/Microsoft.Azure.WebJobs.Host.FunctionalTests/Queues/QueueTests.cs

```diff
@@ -175,6 +175,60 @@ public void InvokeWithBindingData()
             Assert.Equal("123", msgs[0].AsString);
         }
 
+        public class ProgramWithTriggerAndCompoundBindingData
+        {
+            public class Poco
+            {
+                public SubOject prop1 { get; set; }
+                public string xyz { get; set; }
+            }
+
+            public class SubOject
+            {
+                public string xyz { get; set; }
+            }
+
+
+            // BindingData is case insensitive. 
+            // And queue name is normalized to lowercase. 
+            public const string QueueOutName = "qName-{prop1.xyz}";
+            public void Func([QueueTrigger(QueueName)] Poco triggers, [Queue(QueueOutName)] ICollector<string> q)
```

Comment:
discussed in person. Going the 'subobject prop1' approach. Added to tests. 

---
In reply to: [114880093](https://github.com/Azure/azure-webjobs-sdk/pull/1117#discussion_r114880093) [](ancestors = 114880093,114873333)